tests/storage/storage.go

```diff
@@ -145,6 +145,13 @@ var _ = SIGDescribe("Storage", func() {
 				pv  *k8sv1.PersistentVolume
 			)
 
+			cleanUp := func(vmi *virtv1.VirtualMachineInstance) {
+				By("Cleaning up")
+				err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Delete(context.Background(), vmi.ObjectMeta.Name, &metav1.DeleteOptions{})
+				Expect(err).ToNot(HaveOccurred(), failedDeleteVMI)
+				libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 180)
+			}
```

Comment:
You probably do not need to manually clean up the VM. It should be done automatically after each test.

Reply:
@vasiliy-ul we actually do need, because otherwise the [removal of the scsi_debug module](https://github.com/kubevirt/kubevirt/blob/main/tests/storage/storage.go#L157) can fail if the device is still in use. I can maybe add a comment to explain why

Reply:
Ah, okay, then probably makes sense to clean up. I think a comment about that would be really beneficial here, as it is not obvious at first look.